src/test/java/api/loans/EndExpiredPatronActionSessionTests.java

```diff
@@ -9,44 +9,79 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
+import api.support.APITests;
+import api.support.builders.EndSessionBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
 import org.awaitility.Awaitility;
-import org.folio.circulation.support.http.client.IndividualResource;
 import org.hamcrest.Matchers;
-import org.junit.Before;
 import org.junit.Test;
 
-import api.support.APITests;
-import api.support.builders.EndSessionBuilder;
-import io.vertx.core.json.JsonObject;
+import org.folio.circulation.domain.notice.session.PatronActionType;
+import org.folio.circulation.support.http.client.IndividualResource;
 
 public class EndExpiredPatronActionSessionTests extends APITests {
 
-  @Before
-  public void initSession() throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+  @Test
+  public void expiredEndSessionAfterCheckOut()
```

Comment:
What is an `expired end session`?

Reply:
this means that we need to end an expired session.  You can find "End session" term in the description of the story [CIRC-560](https://issues.folio.org/browse/CIRC-560)

Reply:
Please can you describe the process or scenario that this test is demonstrating.

Is it demonstrating that check out notices are sent when an expired session is ended?

Reply:
This test demonstrates that we delete user session records from user-session table if a session is expired. 
The case `when check out notices are sent if an expired session`  is covered in PatronActionSessionTests class